qa/suites/fs/functional/tasks/mds-full.yaml

```diff
@@ -12,6 +12,8 @@ overrides:
       - is full \(reached quota
       - POOL_FULL
       - POOL_BACKFILLFULL
+      - PG_RECOVERY_FULL
+      - PG_DEGRADED
```

Comment:
`PG_DEGRADED` necessary?

Reply:
From the ceph doc: https://docs.ceph.com/en/latest/rados/operations/health-checks/, there has two reasons that will cause this:

```
has the degraded or undersized flag set, meaning there are not enough instances of that placement group in the cluster;

has not had the clean flag set for some time.
```

After filling the pool, it may cause the pgs in peering state for a while, which could make the `clean` flag not set for some time.

From my test I can reproduce it almost 100%.